pages/about/citing.md

```diff
@@ -0,0 +1,33 @@
+layout: page
+title: Citing
+permalink: /citing
+nav_order: 6
+parent: About
```

Comment:
I think it would be better to have this as a top-level entry, just to make sure it is clearly visible.

Reply:
If it is under about, you need to click on about to unfold it and see it.

Reply:
OK, fair enough. I have no strong opinion. Moved to the top level.